client/payment-details/summary/index.tsx

```diff
@@ -102,6 +111,11 @@ const PaymentDetailsSummary = ( {
 		: placeholderValues;
 	const renderStorePrice =
 		charge.currency && balance.currency !== charge.currency;
+	const { authorization } = useAuthorization(
+		'auth_1234',
```

Comment:
@mgascam Are we supposed to get authorization id from page query params? If yes, we'll probably need to ensure it's passed to Payment Details page from Order Details and Transactions pages.

Reply:
> @mgascam Are we supposed to get authorization id from page query params? If yes, we'll probably need to ensure it's passed to Payment Details page from Order Details and Transactions pages.

Thanks @anurag, I'll make a note to check it.

Reply:
@anu-rock on second thought, I decided to take a different approach here: I am using the `payment_intent_id` instead of the `authorization_id` (check [63c2645](https://github.com/Automattic/woocommerce-payments/pull/4688/commits/63c26452bbceacde88c580ba59da17aadcafae57))

1. The new charges table includes the payment_intent_id column, and it has an index for it.
2. The `payment_intent_id` is already available in the Payment Details page so no modifications needed.
3. In the transactions list we have access to both so it is not a big deal to use `payment_intent_id`

I believe you are quite busy at the moment but just wanted to let you know and hear your opinion in case you have any comments.
Thanks!

Reply:
@mgascam That's a good idea!